lesson2/task5.js

```diff
@@ -0,0 +1,18 @@
+5.Реализовать четыре основные арифметические операции в виде функций с двумя параметрами.
+ Обязательно использовать оператор return.
+
+ function Sum(a,b){
```

Comment:
Не стоит функциям давать имена с заглавной буквы. Мы скоро дойдем до ООП и узнаем, что такое поведение обычно используется для классов. Так же, не забываем ставить пробелы между операторами и операндами
```suggestion
 function Sum(a, b){
```